.css-lint.yml

```diff
@@ -0,0 +1,20 @@
+scss_files: 'src/static/styles/**/*.css'
+exclude: 'src/static/styles/vendor/**'
+
+linters:
```

Comment:
`scss-lint` does not support merging of configuration files. We could maybe write something that does this for us, but this is a much simpler way right now - even with the duplication.